vm/stack_internal_test.go

```diff
@@ -16,24 +16,69 @@
 
 package vm
 
-import "testing"
+import (
+	"testing"
+
+	"github.com/magiconair/properties/assert"
+)
 
 func TestStack_new(t *testing.T) {
+	stack := newStack()
+	assert.Equal(t, stackMaxSize, cap(stack.items))
```

Comment:
Someone(including me) uses t.Fatalf method.
I think it is good to use one method to print error.
Or, is there any reason for using assert?